source/extensions/filters/listener/tls_inspector/tls_inspector.h

```diff
@@ -98,6 +101,9 @@ class Filter : public Network::ListenerFilter, Logger::Loggable<Logger::Id::filt
   uint64_t read_{0};
   bool alpn_found_{false};
   bool clienthello_success_{false};
+  // We dynamically adjust the number of bytes requested by the filter up to the
+  // maxConfigReadBytes.
+  uint32_t requested_read_bytes_{0};
```

Comment:
The default value should probably be more than 0 (as it is doubled).
As this is being overridden in the c'tor, I think the initialization here should be removed.

Reply:
done.